src/app/my-donations/my-donations.component.html

```diff
@@ -16,11 +16,23 @@
 
           <div class="donation-container">
             @for (donation of donations; track donation.transactionId) {
+              <biggive-container-card
+                card-colour="tertiary"
+                background-colour="primary"
+              >
                 <div class="donation">
-                  <a href="/campaign/{{donation.projectId}}">{{ donation.charityName }}</a> - {{ donation.createdTime | date: 'mediumDate' }} <br/>
-                  {{ donation.donationAmount | exactCurrency:donation.currencyCode }} ({{this.displayMethodType(donation)}}) <br/>
-                  Reference: {{ donation.transactionId }}
+                  <p>{{this.displayMethodType(donation)}}</p>
+                  <h3><a href="/campaign/{{donation.projectId}}">{{ donation.charityName }}</a></h3>
```

Comment:
Currently only displaying the charity name not the campaign name because the data sent from matchbot doesn't give the campaign name. But it should be very easy to add it to what matchbot sends, and I think that's worth doing. Not necessarily in this PR since I don't want to make the frontend deploy have to wait behind a matchbot deploy.